dev/io.openliberty.checkpoint/src/io/openliberty/checkpoint/internal/openj9/ExecuteCRIU_OpenJ9.java

```diff
@@ -59,8 +61,10 @@ public void dump(Runnable prepare, Runnable restore, File imageDir, String logFi
             throw new CheckpointFailedException(Type.JVM_CHECKPOINT_FAILED, e.getMessage(), e);
         } catch (SystemCheckpointException e) {
             throw new CheckpointFailedException(Type.SYSTEM_CHECKPOINT_FAILED, e.getMessage(), e);
-        } catch (RestoreException e) {
+        } catch (JVMRestoreException e) {
```

Comment:
We may need to do this in the catch for `RuntimeException e` below such that we can stage this in to work on the current EA OpenJ9 build as well as the new one.  Something like:

```java
} catch (RuntimeException e) {
    String exceptionName = e.getClass().getSimpleName();
    if ("JVMRestoreException".equals(exceptionName) || "RestoreException".equals(exceptionName)) {
        throw new CheckpointFailedException(Type.JVM_RESTORE_FAILED, e.getMessage(), e);
    }
    if ("SystemRestoreException".equals(exceptionName)) {
        throw new CheckpointFailedException(Type.SYSTEM_RESTORE_FAILED, e.getMessage(), e);
    }
    throw new CheckpointFailedException(checkpointImpl.getUnknownType(), e.getMessage(), e);
}
```
Then once we have an EA with the API change and we have moved all the infrastructure to the new Open J9 EA build then we can move back to actual catch blocks with the new exceptions.